lib/truffle/java.rb

```diff
@@ -0,0 +1,50 @@
+###### BEGIN LICENSE BLOCK ######
```

Comment:
I think we could flatten this a bit, and it could go into `lib/java`, without `truffle` in the middle.